irc.html

```diff
@@ -13,21 +13,21 @@ <h1>Channels</h1>
   </p>
   <ul>
     <li>
-      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#Minetest</a></strong>
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel where all players and contributors can discuss the game or request support.
```

Comment:
why explicitly name which group of people this channel for?

Reply:
It was something we came with when discussing about it on #-dev. Do you have other suggestions?